deploy/k8s/manifest.yaml

```diff
@@ -1851,6 +1851,10 @@ spec:
           envFrom:
```

Comment:
wasn't k8s manifest dropped?

Reply:
yes in other PR that isn't merged yet https://github.com/solarwinds/swi-k8s-opentelemetry-collector/pull/130 (there are some actions that needs to be done before merging that)